modules/seo-tools/jetpack-seo.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * An SEO expert walks into a bar, bars, pub, public house, Irish pub, drinks, beer, wine, liquor, Grey Goose, Cristal...
+ */
+class Jetpack_SEO {
+	public function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	public function init() {
+		if ( apply_filters( 'jetpack_seo_meta_tags', true ) ) {
+			add_action( 'wp_head', array( $this, 'meta_tags' ) );
+
+			// Add support for editing page excerpts in pages, regardless of theme support.
+			add_post_type_support( 'page', 'excerpt' );
+		}
+
+		if ( apply_filters( 'jetpack_seo_custom_titles', true ) ) {
+			// Overwrite page title with custom SEO meta title for themes that support title-tag.
+			add_filter( 'pre_get_document_title', array( 'Jetpack_SEO_Titles', 'get_custom_title' ) );
+
+			// Add overwrite support for themes that don't support title-tag.
+			add_filter( 'wp_title', array( 'Jetpack_SEO_Titles', 'get_custom_title' ) );
+		}
+
+		add_filter( 'jetpack_open_graph_tags', function( $tags ) {
+			$custom_title = Jetpack_SEO_Titles::get_custom_title();
+			$post_custom_description = Jetpack_SEO_Posts::get_post_custom_description( get_post() );
+
+			if ( ! empty( $custom_title ) ) {
+				$tags['og:title'] = $custom_title;
+			}
+
+			if ( !empty ( $post_custom_description ) ) {
+				$tags['og:description'] = $post_custom_description;
+			}
+
+			return $tags;
+		} );
+	}
+
+	private function get_authors() {
+		global $wp_query;
+
+		$authors = array();
+
+		foreach ( $wp_query->posts as $post ) {
+			$authors[] = get_the_author_meta( 'display_name', (int) $post->post_author );
+		}
+
+		$authors = array_unique( $authors );
+
+		return $authors;
+	}
+
+	function meta_tags() {
+		global $wp_query;
+
+		$period = $template = '';
+		$meta = array();
+		$conflicted_themes = apply_filters( 'jetpack_seo_meta_tags_conflicted_themes', array() );
+
+		if ( isset( $conflicted_themes[ get_option( 'template' ) ] ) ) {
+			return;
+		}
+
+		$front_page_meta = Jetpack_SEO_Utils::get_front_page_meta_description();
+		$description = $front_page_meta ? $front_page_meta : get_bloginfo( 'description' );
+
+		$site_host = apply_filters( 'jetpack_seo_site_host', 'WordPress' );
```

Comment:
Missing docblock here.

I wonder if that filter should give an empty string by default, as I don't think self-hosted WordPress sites would want to include that in their Meta title, since it doesn't give useful information about the site itself.